Documentation/articles/getting_started/toc.yml

```diff
@@ -4,11 +4,15 @@
     href: 1_creating_a_new_project_vs.md
   - name: Visual Studio for Mac
     href: 1_creating_a_new_project_vsm.md
-  - name: DotNet core CLI
+  - name: .NET Core CLI

```

Comment:
.NET CLI, not .NET Core CLI

Reply:
It is .NET Core CLI: https://docs.microsoft.com/en-us/dotnet/core/tools/
Though probably will be just .NET CLI when .NET 5 releases.